pkgs/top-level/all-packages.nix

```diff
@@ -41686,6 +41686,8 @@ with pkgs;
 
   ssl-proxy = callPackage ../tools/networking/ssl-proxy { };
 
+  ssg-cli = with python3Packages; toPythonApplication ssg;
```

Comment:
I'm not sure if this warrants adding another attribute with a different name to the package list just to make the executable available. We can achieve this by just including `meta.mainPackage` instead.